README.md

```diff
@@ -1,6 +1,6 @@
 # Azure Functions for Visual Studio Code (Preview)
 
-## Prerequisites
+## Prerequisites for local debugging
 * [Node v8.0+](https://nodejs.org/)
   * Older versions of node will be supported soon. See issue [#1](https://github.com/Microsoft/vscode-azurefunctions/issues/1)
 * [.NET Core 2.0](https://www.microsoft.com/net/download/core)
```

Comment:
Are you sure this is true?  I had thought apps were supposed to install .net core libs with their app.

Reply:
Yep: https://docs.microsoft.com/azure/azure-functions/functions-run-local

They're _supposed_ to bundle .net core libs with their app, but they haven't done that quite yet (After all, the cli is still technically in preview)